### ISSUE TITLE
entity/relation embedding normalization?

### ISSUE THREAD

**Issue by ShomyLiu** [thunlp/TensorFlow-TransX#9]
It seems that there is no norm for the embedding. So I wonder whether the performance can be as good as the paper's result

**Comment by THUCSTHanxu13** [comment]
From our experiment results on WN18 and FB15K, non-normalized models can perform as well as normalized models and be trained more efficient. You can try to compare these models on other data sets.
